go/vt/vttablet/tabletserver/connpool/pool.go

```diff
@@ -305,6 +305,15 @@ func (cp *Pool) Active() int64 {
 	return p.Active()
 }
 
+// UsagePercent returns the connection pool usage as a percentage
+func (cp *Pool) UsagePercent() float64 {
+	p := cp.pool()
+	if p == nil {
+		return 0
+	}
+	return (float64(p.Active()) / float64(p.Capacity()) * 100)
+}
+
```

Comment:
inUse is the metric that should be used instead of active.
active means open MySQL connections
inuse means currently busy connections.